samples/solidity/contracts/InterfaceCheck.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import './IERC1155MixedFungible.sol';
+
+/**
+ * Test utility for checking ERC165 interface identifiers.
+ */
+contract InterfaceCheck {
```

Comment:
After looking more at the connector, it seems to me we don't even need to support ERC165. If I am understanding the code currently, the 1155 connector does not allow creating pools from existing contracts on chain.